### PR TITLE
Fixes #1188 - Educate developers on type input feature

### LINKED ISSUE #1188: Educate developers on type input feature

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

**Comment by Simpcyclassy** [comment]
image

**Comment by Simpcyclassy** [comment]
This is still a work in progress. I will notify you when I need further review just wanted to recreate the PR. Again can the PR labels be made available @ianb

**Comment by Simpcyclassy** [comment]
image
Updated version of feature

**Comment by Simpcyclassy** [comment]
@ianb could you look at my Pr please

File: extension/popup/popupView.jsx [hunk 1]
```diff
@@ -480,8 +480,18 @@ const VoiceInput = ({ suggestions, onClickLexicon }) => {
   };
   return (
     <div id="voice-input">
-      {suggestions ? (
+      {buildSettings.inDevelopment ? (
```
Comment (ianb) [inline extension/popup/popupView.jsx]: It looks like this is going to block suggestions except in development.

The `{suggestions ? ...` part needs to stay

File: extension/popup/popupView.jsx [hunk 2]
```diff
@@ -480,8 +480,18 @@ const VoiceInput = ({ suggestions, onClickLexicon }) => {
...
+          <p>Developer tip:</p>
+          <a
+            target="_blank"
+            rel="noopener"
+            id="lexicon"
+            href="../views/lexicon.html"
+            onClick={onInputStarted}
+          >
+            Start typing to make a request using a keyboard
+          </a>
```
Comment (ianb) [inline extension/popup/popupView.jsx]: Instead put a `<div>` around this new message, and put the conditional around that, like `{buildSettings.inDevelopment ? <div><p>Developer tip:...</div> : null}`

File: extension/popup/popupView.jsx (hunk 1 above)
Comment (Simpcyclassy) [reply extension/popup/popupView.jsx]: Alright, fixing that now.

**Comment by Simpcyclassy** [comment]
You are welcome. I am opportuined!

**Pull request merged**